.eslintrc

```diff
@@ -0,0 +1,4 @@
+extends: "xo-space/esnext"
```

Comment:
It came with `generator-generator`, and I didn't see a reason to change it. We totally can though.